test/include/executioners/SteadyWithPicardCheck.h

```diff
@@ -26,11 +26,6 @@ class SteadyWithPicardCheck : public Steady
    */
   virtual void preSolve() override;
 
-  /**
-   * Additional convergence check to be honored by PicardSolve
-   */
-  virtual bool augmentedFixedPointConvergenceCheck() const override;
-
```

Comment:
What's the reason why this had to be removed?

Reply:
Nobody is using it.  I asked on the mailing list and scoured app repositories.  Griffin folks added it originally because they thought it would help with some stuff - but it didn't and now they don't use or need it.

Reply:
👍

Reply:
I removed it in Griffin for my proposed PR in MOOSE. We do not need it.